go-controller/pkg/util/ovs.go

```diff
@@ -149,6 +149,26 @@ func SetExec(exec kexec.Interface) error {
 	return nil
 }
 
+// SetSelectedExec validates executable paths for selected commands. It also saves the given
+// exec interface to be used for running selected commands
+func SetSpecificExec(exec kexec.Interface, commands []string) error {
```

Comment:
@girishmg how about `SetSpecificExec(exec kexec.Interface, commands ...string)` ? Then you don't need to do []string{"ovs-vsctl"} in the callers.

Reply:
@dcbw done. Thanks.